token-allocation/src/adapter.ts

```diff
@@ -1,31 +1,34 @@
 import { AdapterResponse, Execute, AdapterRequest } from '@chainlink/types'
 import { Requester, Validator } from '@chainlink/external-adapter'
-import Decimal from 'decimal.js'
-import { utils } from 'ethers'
 import { Config, makeConfig } from './config'
+import { Allocations } from './types'
+import { Decimal } from 'decimal.js'
 
 export type Index = IndexAsset[]
 
 type IndexAsset = {
```

Comment:
Do we even need this type anymore?

I think this a relic of defi-pule and looks like a duplicate of `TokenAllocation` now.

Reply:
This is basically the response schema now.